docs/_packages/socket_mode.md

```diff
@@ -194,7 +195,8 @@ A very simple custom logger might ignore the name and level, and write all messa
 const { createWriteStream } = require('fs');
 const logWritable = createWriteStream('/var/my_log_file'); // Not shown: close this stream
 
-const socketModeClient = new SocketModeClient(appToken, {
+const socketModeClient = new SocketModeClient({
```

Comment:
SocketModeClient does not table app-level token in the same way with WebClient.

Reply:
Good catch!